torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -912,7 +922,7 @@ RegisterOperators reg(
            // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
            int64_t a, b;
            pop(stack, a, b);
-           push(stack, pow(a, b));
+           push(stack, powWrapper(a, b));
```

Comment:
@zhxchen17, removing this operator led to internal tests failure: https://www.internalfb.com/diff/D30183946

I am going to keep this operator as a result.